MaybeGame/Player.cs

```diff
@@ -4,279 +4,86 @@ namespace MaybeGame
 {
     public class Player
     {
-        //TODO: Make a class to house stats and a class for Experience!!!! Make enum races
+        PlayerStats playerStats = new ();
+        PlayerExperience playerExperience = new ();
 
         public string UserName { get; set; }
         public string ClanName { get; set; }
-        public string Race { get; set; }
-        public int Strength { get; set; }
-        public int Stamina { get; set; }
-        public int Agility { get; set; }
-        public int Dexterity { get; set; }
-        public int Health { get; set; }
-        public int Defence { get; set; }
-        public double DodgeChance { get; set; }
-        public double Speed { get; set; }
-        public int AttackPower { get; set; }
-        public double ExperiencePoints { get; set; }
-        //public double SavedExperiencePoints { get; set; }
-        public int StatPoints { get; set; }
-        public int Level { get; set; }
-
+        public PlayerRaces Race { get; set; }
```

Comment:
Races??? Multiracial character??

Reply:
Not completely wrong but they are of different races